scripts/histmakers/w_z_gen_dists.py

```diff
@@ -4,17 +4,20 @@
 
 import narf
 import wremnants
-from wremnants import theory_tools,syst_tools,theory_corrections
+from wremnants import theory_tools,syst_tools,theory_corrections, theoryAgnostic_tools
 from wremnants.datasets.dataset_tools import getDatasets
 import hist
 import math
 import os
+import numpy as np
 
 
 parser.add_argument("--skipAngularCoeffs", action='store_true', help="Skip the conversion of helicity moments to angular coeff fractions")
 parser.add_argument("--singleLeptonHists", action='store_true', help="Also store single lepton kinematics")
 parser.add_argument("--skipEWHists", action='store_true', help="Also store histograms for EW reweighting. Use with --filter horace")
 parser.add_argument("--absY", action='store_true', help="use absolute |Y|")
+parser.add_argument("--ptqVgen", action='store_true', help="If want to store qt by q")
+parser.add_argument("--addhelicity", action='store_true', help="If want to store qt by q")
```

Comment:
I have the impression that you didn't pull from the upstream, your code misses a similar option that was already merged in the central code (--addHelicityToPDFs).
Make sure to use "git pull --recurse-submodules upstream main" to update your PR (there are also a couple of conflicts at the moment, you should fix them in order to trigger CI)